In house patch to correctly identify large file support in a 64-bit
compilation environment.
A correction of this sort should be sent upstream.
--- perl-5.36.0/hints/solaris_2.sh
+++ perl-5.36.0/hints/solaris_2.sh
@@ -495,9 +495,9 @@ case "$uselargefiles" in
 ''|$define|true|[yY]*)
 
 # Keep these in the left margin.
-ccflags_uselargefiles="`$run getconf LFS_CFLAGS 2>/dev/null`"
-ldflags_uselargefiles="`$run getconf LFS_LDFLAGS 2>/dev/null`"
-libswanted_uselargefiles="`$run getconf LFS_LIBS 2>/dev/null|sed -e 's@^-l@@' -e 's@ -l@ @g'`"
+ccflags_uselargefiles="`$run getconf LFS64_CFLAGS 2>/dev/null`"
+ldflags_uselargefiles="`$run getconf LFS64_LDFLAGS 2>/dev/null`"
+libswanted_uselargefiles="`$run getconf LFS64_LIBS 2>/dev/null|sed -e 's@^-l@@' -e 's@ -l@ @g'`"
 
     ccflags="$ccflags $ccflags_uselargefiles"
     ldflags="$ldflags $ldflags_uselargefiles"
